### DISCUSSION TITLE
Should those of us who contributed in the pycon sprint still be contributors, or should we go through the fork process?

### DISCUSSION THREAD

**Discussion by machallboyd** [NREL/mappymatch#110] [General]
Would like to keep contributing but I feel like I might have a little too much permissions on the repo like this.

**Comment by nreinicke** [comment]
Yeah you bring up a good point. We've chatted about it internally and are planning to move to a more standard forking workflow.

**Comment by nreinicke** [comment]
@machallboyd @Rowlando13 @grahamwaters - I've started shifting our workflow and removing direct access but have left you on since you have open pull requests. Would you mind moving to the new workflow at your convenience? Once you're able to switch and we close the open pull requests I can finalize the transition. Thank you for helping us out and bearing with us as we navigate the open source space for the first time!

**Reply by machallboyd** [reply]
Can do.

**Reply by Rowlando13** [reply]
First, I am happy to move to this workflow. 
I come from a little different position on this. I think the contributor position / permissions are good for the project to have. It lets someone else help out with admin, and other miscellaneous stuff to keep the project happy and the core maintainers time protected. I think it would be nice to keep on 1-3 contributors. If the decision at this point is to pull contributor permissions wholesale, I would encourage you to think about bringing them back.

**Reply by nreinicke** [reply]
Yeah that's a good perspective. I'm open to having a select few external collaborators with more access that could assist with general project admin (closing requests, removing state branches, etc). @machallboyd @Rowlando13 since you've expressed interest in continuing contributing to this project are either / both of you interesting in taking on that role?

**Reply by Rowlando13** [reply]
I would be happy to!

**Reply by grahamwaters** [reply]
Sounds good to me. I wondered, does it show anywhere that I worked with you guys on our sprint? I am pretty new to this GitHub ecosystem. @nreinicke.

**Comment by machallboyd** [comment]
I would also be happy to.

**Comment by nreinicke** [comment]
Great!! Thank you! Looking forward to continued collaboration
